sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -33,11 +33,11 @@
 /**
  * Catalog methods for working with Tables.
  * <p>
- * TableCatalog implementations may be case sensitive or case insensitive. Spark will pass
+ * TableCatalog implementations may be case-sensitive or case-insensitive. Spark will pass
```

Comment:
These seem cosmetic and unrelated. I'd probably not mix them with adding `TableInfo`.

Reply:
It bothered me and I didn't want to raise a PR to fix this.